lib/ios-deploy.js

```diff
@@ -7,7 +7,7 @@ import log from './logger';
 
 const APPLICATION_INSTALLED_NOTIFICATION = 'com.apple.mobile.application_installed';
 const INSTALLATION_STAGING_DIR = 'PublicStaging';
-const APPLICATION_PUSH_TIMEOUT = 60 * 1000;
+const APPLICATION_PUSH_TIMEOUT = 30 * 1000;
```

Comment:
shouldn't sequential push be slower than the parallel one?

Reply:
This is per file now